MobiFlight/Config/MultiplexerDriver.cs

```diff
@@ -109,10 +103,9 @@ override public bool FromInternal(String value)
                 pins[1] = paramList[2];
                 pins[2] = paramList[3];
                 pins[3] = paramList[4];
-                res = Initialize(pins);
-            } else {
-                _initCounter++;
+                res = SetPins(pins);
             }
+            registerClient();
```

Comment:
I would say, we make this either called from the outside (not my favorite) or only from here (my favorite)

Reply:
As mentioned in other replies (https://github.com/MobiFlight/MobiFlight-Connector/pull/1119#discussion_r1097854556), I find it hard to imagine how we could get rid of the explicit call (from the outside) in the GUI; and for consistency I'd also be inclined to "extract" it from here (in the only remaining call from `Config`.)